content/python/concepts/dates/terms/now/now.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.now()'
+Description: 'Returns the current date and timestamp.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.now()` method of the Python `datetime` module returns the current local date and timestamp as a new `datetime` instance.
```

Comment:
Is it `.now()` or `.datetime.now()`. Should also specify the return type and properties, or link out to `datetime`

Reply:
Oh! Thanks for the catch 😄